src/QualificationRunner.Core/Services/QualificationEngine.cs

```diff
@@ -76,13 +75,13 @@ private async Task<QualificationRunResult> execute(QualifcationConfiguration qua
       {
          _logger.AddDebug(Logs.StartingQualificationRunForProject(qualifcationConfiguration.Project));
 
-         var logFile = Path.Combine(qualifcationConfiguration.TempFolder, "log.txt");
+         var logFilePaths = new List<string> {Path.Combine(qualifcationConfiguration.TempFolder, "log.txt"), runOptions.LogFile } ;
          var configFile = Path.Combine(qualifcationConfiguration.TempFolder, "config.json");
          var project = qualifcationConfiguration.Project;
          var qualificationRunResult = new QualificationRunResult
          {
             ConfigFile = configFile,
-            LogFile = logFile,
+            LogFilePaths = logFilePaths,

```

Comment:
@georgeDaskalakis  I would say here, we only want to save the project specific log file so that the user can check the problem for the one project only if it happens

Reply:
done